packages/plugin-odoo-snippets/src/OdooBindings.ts

```diff
@@ -0,0 +1,163 @@
+import JWEditor from '../../core/src/JWEditor';
+import { VElement } from '../../core/src/VNodes/VElement';
+import { RelativePosition } from '../../core/src/VNodes/VNode';
+import { Char, InsertHtmlParams } from '../../plugin-char/src/Char';
+import { EmptyParams, WrapParams, ReplaceParams } from './OdooSnippet';
+import { ContainerNode } from '../../core/src/VNodes/ContainerNode';
+import { DomLayoutEngine } from '../../plugin-dom-layout/src/ui/DomLayoutEngine';
+import { Layout } from '../../plugin-layout/src/Layout';
+import {
+    AddClassParams,
+    MoveParams,
+    OdooSnippet,
+    RemoveClassParams,
+    RemoveParams,
+    SetAttributeParams,
+    SetStyleParams,
+    ToggleClassParams,
+} from './OdooSnippet';
+
+interface ExectCommandHelpers {
+    [key: string]: (...args: any[]) => void;
+}
+
+export type HtmlPoint = [Node, RelativePosition];
```

Comment:
I'm pretty sure this already exists somewhere under a different name. `DomPoint` maybe ?

Reply:
Couldn't find it when I grepped `, RelativePosition]`